src/11_args.py

```diff
@@ -42,6 +51,9 @@
 # Google "python keyword arguments".
 
 # YOUR CODE HERE
+def f4(**args):
+    for key in args:
+        print(f'key: {key}, value: {args[key]}')
```

Comment:
I like how you used the f string! Another way you could write this in Python is using the items() method. There's also a values() method.
```
def f4(**kwargs):
    for key, value in kwargs.items():
        print(key, value)
```